jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/JettyIntegrationTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxscraper.target_systems;
+
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertGauge;
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertGaugeWithAttributes;
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertSumWithAttributes;
+import static io.opentelemetry.contrib.jmxscraper.target_systems.MetricAssertions.assertSumWithAttributesMultiplePoints;
+
+import io.opentelemetry.contrib.jmxscraper.JmxScraperContainer;
+import java.time.Duration;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+public class JettyIntegrationTest extends TargetSystemIntegrationTest {
+
+  @Override
+  protected GenericContainer<?> createTargetContainer(int jmxPort) {
+    GenericContainer<?> container =
+        new GenericContainer<>(
+            new ImageFromDockerfile()
+                .withDockerfileFromBuilder(
+                    builder ->
+                        builder
+                            .from("jetty:11")
+                            .run(
+                                "java",
+                                "-jar",
+                                "/usr/local/jetty/start.jar",
+                                "--add-to-startd=jmx,stats,http")
+                            .run("mkdir -p /var/lib/jetty/webapps/ROOT/")
+                            .run("touch /var/lib/jetty/webapps/ROOT/index.html")
+                            .build()));
+
+    container
+        .withEnv("JAVA_OPTIONS", genericJmxJvmArguments(jmxPort))
+        .withStartupTimeout(Duration.ofMinutes(2))
+        .waitingFor(Wait.forLogMessage(".*Started Server.*", 1));
+
+    return container;
+  }
+
+  @Override
+  protected JmxScraperContainer customizeScraperContainer(JmxScraperContainer scraper) {
+    return scraper.withTargetSystem("jetty");
+  }
+
+  @Override
+  protected void verifyMetrics() {
+    waitAndAssertMetrics(
+        metric ->
+            assertSumWithAttributes(
+                metric,
+                "jetty.session.count",
+                "The number of sessions established in total.",
+                "{sessions}",
```

Comment:
I agree with you here, it should be `{session}` and not `{sessions}`. I think however that it would be easier to leave it as-is to ensure maximum compatibility, maybe with a comment about this inconsistency as an attempt not to reproduce it in the future.

Reply:
Yeah, it concerns me that we'll never be able to fix this if we perpetuate it. Any ideas on when/how we might do that?

Reply:
fixed in 187ca89